code/mobility-services/mobility-services.css

```diff
@@ -228,6 +237,53 @@ a.big-button-container {
 a.small-button {
   text-decoration: none;
 }
+/****************************************/
+/*** Dropdown Menu Buttons Navigation ***/
+/****************************************/
+
+/* hide custom menu if on mobile */
+@media (max-width: 799px) {
+  #tia-menu-list {
```

Comment:
tia reference not updated. I generically call it my desktop-menu-list now instead of specifying app/module name so its app agnostic and we wont have to change it if adding to other apps